restler/unit_tests/log_baseline_test_files/Bug_Buckets_Json/InvalidValueChecker_500_1.json

```diff
@@ -0,0 +1,17 @@
+{
+    "status_code": "500",
+    "checkerName": "InvalidValueChecker",
+    "reproducible": true,
+    "verb": "PUT",
+    "endpoint": "/item/{itemName}",
+    "status_text": "Internal Server Error",
+    "request_sequence": [
+        {
```

Comment:
We discussed earlier that you'd also test with multiple items in the request sequence - did you have issues with that example, or is this still WIP?

Reply:
I was trying to get a test with multiple items , I was not able to find one. The test_quick_start.py was able to create with multiple items but that was using the demoserver in local host , but the unit_tests were using unittest server that did create multiple items.

Reply:
The 'test_checkers' test produces bugs with multiple items (see 'checkers_bug_buckets.txt').  You do not need to verify all those individual bug buckets files in the json format, but it would be great if you could pick one file with multiple items, plus the bug_buckets.json file and check them in as baselines.  That way we can easily see modifications in PRs.